protectli/vault_bsw/BraswellFspBinPkg/Include/FspUpdVpd.h

```diff
@@ -0,0 +1,539 @@
+/** @file
+
+Copyright (c) 2017, Intel Corporation. All rights reserved.<BR>
```

Comment:
@miczyg1 Do we include it in the end, even if we said we can use the one from official repository?

Reply:
Yes, because it is the official public one. Like the README says it is just public FSP binary patched to have the correct PMIC values for FW4C